components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/APIUsageStatisticsClient.java

```diff
@@ -977,9 +978,9 @@ private String buildKeySetString(String[] keyArray){
     public List<APIVersionUsageDTO> getUsageByAPIVersions(String providerName, String apiName,
                                                           String fromDate, String toDate) throws APIMgtUsageQueryServiceClientException {
 
-        OMElement omElement = this.queryBetweenTwoDaysForAPIUsageByVersion(
+        List<APIUsage> usageData = this.queryBetweenTwoDaysForAPIUsageByVersion(
                 APIUsageStatisticsClientConstants.API_VERSION_USAGE_SUMMARY, fromDate, toDate, apiName);
-        Collection<APIUsage> usageData = getUsageData(omElement);
+//        Collection<APIUsage> usageData = getUsageData(omElement);
```

Comment:
content is replace in commit: https://github.com/wso2/carbon-apimgt/commit/078e20af796c5e61f8f9696b995e05cb82ff8953#diff-bb18fa9c4e9abc621195a0ecc75bef6d